client/app/services/usage-graphs/usage-graphs.service.js

```diff
@@ -0,0 +1,79 @@
+/** @ngInject */
+export function UsageGraphsFactory() {
+  var service = {
+    cpuChartConfig: getCPUChartConfig,
+    memoryChartConfig: getMemoryChartConfig,
+    storageChartConfig: getStorageChartConfig,
+    convertBytestoGb: convertBytestoGb,
+  };
+
+  function getCPUChartConfig(used, total) {
+    let usedValue = 0;
+    let totalValue = 0;
+
+    if (angular.isDefined(used)) {
+      usedValue = used;
+      totalValue = total;
+    }
+
+    return {
```

Comment:
Reworked this service to have a single function that takes care of building up the config.